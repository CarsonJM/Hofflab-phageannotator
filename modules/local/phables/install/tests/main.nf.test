nextflow_process {

    name "Test Process: PHABLES_INSTALL"
    script "../main.nf"
    process "PHABLES_INSTALL"

    test("No input required") {

        then {
            assertAll (
                { assert process.success },
                { assert snapshot(process.out.phables_db).match() }
            )
        }
    }

    test("No input required - stub") {

        options "-stub"

        then {
            assertAll (
                { assert process.success },
                { assert snapshot(process.out.phables_db).match() }
            )
        }
    }
}
