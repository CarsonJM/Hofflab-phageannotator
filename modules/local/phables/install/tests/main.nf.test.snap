{
    "No input required": {
        "content": [
            [
                [
                    "marker.hmm:md5,3b63057612a5512871dd3a0e158af824",
                    "phrog_annot_v4.tsv:md5,3f078c589efeef3352c5e60c1ced9a44",
                    [
                        "phrogs_db:md5,75649b7e711af909288b94ebaec8e724",
                        "phrogs_db.dbtype:md5,c31b034959815c44d4dafd6f30846159",
                        "phrogs_db.index:md5,cd1b2517d80797c8e2fb0f6666482a1d",
                        "phrogs_profile_db:md5,d355c1b2599c9fa1a2e19b587cc3e75a",
                        "phrogs_profile_db.dbtype:md5,f2dd0dedb2c260419ece4a9e03b2e828",
                        "phrogs_profile_db.index:md5,61d1b0e42f135379b553c8f48372ca44",
                        "phrogs_profile_db_consensus:md5,4107609c2701b2eca2f561177d1498f3",
                        "phrogs_profile_db_consensus.dbtype:md5,f1d3ff8443297732862df21dc4e57262",
                        "phrogs_profile_db_consensus.index:md5,a711c8e652b63aa2aceb20716ae8c600",
                        "phrogs_profile_db_h:md5,25c9cbd7ac212a198f6da1e1883ea5b9",
                        "phrogs_profile_db_h.index:md5,ae5e93f21c408190ea41e1bbf1b10ff4",
                        "phrogs_profile_db_seq:md5,566a77a95040aeed03d72f2d7a369607",
                        "phrogs_profile_db_seq.dbtype:md5,f1d3ff8443297732862df21dc4e57262",
                        "phrogs_profile_db_seq.index:md5,a711c8e652b63aa2aceb20716ae8c600",
                        "phrogs_profile_db_seq_h:md5,25c9cbd7ac212a198f6da1e1883ea5b9",
                        "phrogs_profile_db_seq_h.index:md5,ae5e93f21c408190ea41e1bbf1b10ff4"
                    ],
                    "phrogs_mmseqs_db.tar.gz:md5,7cdeee285789a73ad836829f13311669"
                ]
            ]
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "24.04.1"
        },
        "timestamp": "2024-07-10T16:56:18.925676399"
    },
    "No input required - stub": {
        "content": [
            [
                "/mmfs1/gscratch/pedslabs_hoffman/carsonjm/CarsonJM/HoffLab-phageannotator/.nf-test/tests/af218e93d145f4972d4106c6195acbf4/work/36/b7c429b8f4aa2c36460cd1eadec21b/phables_db"
            ]
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "24.04.1"
        },
        "timestamp": "2024-07-10T16:56:37.111419554"
    }
}